Library/Homebrew/formula.rb

```diff
@@ -1455,16 +1455,6 @@ def system(cmd, *args)
       $stdout.flush
 
       unless $?.success?
-        log_lines = ENV["HOMEBREW_FAIL_LOG_LINES"]
-        log_lines ||= "15"
-
-        log.flush
-        if !verbose || verbose_using_dots
-          puts "Last #{log_lines} lines from #{logfn}:"
-          Kernel.system "/usr/bin/tail", "-n", log_lines, logfn
-        end
-        log.puts
-
```

Comment:
Sounds good. @xu-cheng, were you thinking that `HOMEBREW_FAIL_LOG_LINES` would control how many lines to show, rather than always using 15 as we do now?